src/main/java/com/google/sps/classes/Notification.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+import java.sql.Timestamp;
+
+/**
+ * Class to represent notifications and send them to front end as converted JSON.
+ */
+public class Notification {
+  public String message = "";
+  public String url = "";
```

Comment:
Do you ever plan on using notifications without either of these fields initialized? If not, no need to initialize by default.

Reply:
This class was made by Andrés. Should I tell him this, add a TODO, change it myself...?